gulp/config.js

```diff
@@ -1,11 +1,65 @@
-'use strict';
+var dest = "./dist/assets/";
+var src = "./src/assets";
+var version = "/1.0.0";
 
 module.exports = {
+  concatJS: {
+    documentation: {
+      src: [
+        src + '/js/site/global.js',
+        src + '/js/site/prism.js'
+      ],
+      dest: dest + '/js'
+    },
+    protocol: {
+      src: [
+        src + '/js/protocol/*.js'
+      ],
+      dest: dest + '/js'
+    }
+  },
+  lintCss: {
+    src: [
+      dest + "/sass/protocol/protocol.css",
+      dest + "/sass/site/site.css"
```

Comment:
I think these should be `.scss` files, not `.css`? And this would be another place to do the whole folder like `"/sass/**/*.scss"`